expresssession/test/Student.test.js

```diff
@@ -0,0 +1,69 @@
+const mongoose = require('mongoose')
+const chai = require('chai')
+const chaiHttp = require('chai-http')
+
+const { Student } = require('../server/model/student.model')
+
+const server = require('../server/server')
+let should = chai.should();
+
+
+chai.use(chaiHttp);
+
+
+describe('Students' , () => {
+
+    beforeEach((done) => {
+        Student.remove({} , (err)=> {
```

Comment:
Why are we doing this?

Reply:
in order to remove everything from database stuudent however , here we were supposse to use testing database but this demo app so i used the same